main/adv_post.c

```diff
@@ -289,6 +304,7 @@ adv_post_task(void)
 void
 adv_post_init(void)
 {
+    g_adv_post_nonce = esp_random();
```

Comment:
I think the nonce is supposed to be updated for each request and sent along in the header. This prevents replay attacks

Reply:
Nonce gets incremented after post, ok